SPECS/filesystem/filesystem.spec

```diff
@@ -607,7 +605,6 @@ return 0
 %config(noreplace) /etc/fstab
 %config(noreplace) /etc/group
 %config(noreplace) /etc/hosts
-%config(noreplace) /etc/host.conf
```

Comment:
Which package would be responsible for populating the file if we remove it from here ?